deploy/src/main/deploy/kubernetes/azure_deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Absolute path this script is in
+SCRIPTPATH="$(cd "$(dirname "$0")" && pwd -P)"
+RESOURCES=$SCRIPTPATH/../resource-descriptors
+NS=hono
+
+echo DEPLOYING ECLIPSE HONO TO AZURE AKS
+set +e
+source $SCRIPTPATH/olm_deploy.sh
+source $SCRIPTPATH/kubernetes_deploy.sh
+set -e
+
+echo CREATE AZURE EXTERNAL SERVICES
+kubectl delete service hono-dispatch-router-ext -n hono || true
+
+kubectl create -f $RESOURCES/azure --namespace $NS
```

Comment:
my understanding is that you overwrite the service definitions in order to use the LoadBalancer type instead of defining a NodePort, right?

Reply:
Yes. By preparing tutorial and scripts I tried to avoid any changes in original files. That is why I have this "workaround".

Reply:
I see. We have deprecated the script based installation to kubernetes some weeks ago and want to focus on Helm based installation in the future. A next step could therefore be, to incorporate Azure deployment into the existing Helm templates by means of another deployment target variant like we already do for distinguishing between *plain* kubernetes and Openshift. But I think we can do that in a separate PR (if you are interested in doing it).

Reply:
This is something I'm focused on right now.